notebooks/data-sources/TestGrid/metrics/correlated_failures.ipynb

```diff
@@ -0,0 +1,1671 @@
+{
```

Comment:
<p>If this is going to be a standalone notebook, we could copy a few of the markdowns from the original notebook such as for defining "failure groups" and "Encoding".  If we are using this with the old notebook then I guess it's fine...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>